### PR TITLE
Add description to Countereamples.isIso and a link to proposition.

### PR DESCRIPTION
**Pull request by jappeace**
# Description

Add description to counterexample.isiso

## Checklist

Before submitting a merge request, please check the items below:

- [x] The imports are sorted (use `find -type f -name \*.agda -or -name \*.lagda.md | xargs support/sort-imports.hs`)

- [x] All code blocks have "agda" as their language. This is so that
tools like Tokei can report accurate line counts for proofs vs. text.

- [x] Proofs are explained to a satisfactory degree; This is subjective,
of course, but proofs should be comprehensible to a hypothetical human
whose knowledge is limited to a working understanding of non-cubical
Agda, and the stuff your pages link to.

The following items are encouraged, but optional:

- [ ] If you feel comfortable, add yourself to the Authors page! Add a
profile picture that's recognisably "you" under support/pfps; The
picture should be recognisable at 128x128px, should look good in a
squircle, and shouldn't be more than 200KiB.

- [ ] If your contribution makes mention of a negative statement, but
does not prove the negative (perhaps because it would distract from the
main point), consider adding it to the counterexamples folder.

- [ ] If a proof can be done in both "cubical style", and "book HoTT
style", and you have the energy to do both, consider doing both!
However, it is **completely fine** to only do one! For instance, I
(Amélia) am much better at writing proofs "book-style".

### CODE REVIEW

File: src/1Lab/Counterexamples/IsIso.lagda.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+description: |
+    By counter example we show that is-iso is not a proposition.
```
Comment (jappeace) [inline src/1Lab/Counterexamples/IsIso.lagda.md:3]: Doesn't seem to be much else going on :thinking:

File: src/1Lab/Counterexamples/IsIso.lagda.md [hunk 2]
```diff
@@ -13,10 +17,12 @@ module 1Lab.Counterexamples.IsIso where
 
 # is-iso is not a proposition
 
-We show that if `is-iso`{.Agda} were a proposition, then `(x : A) → x ≡
+We show that if `is-iso`{.Agda} were a [proposition], then `(x : A) → x ≡
 x` would be contractible for any choice of `A`. Taking `A` to be
 `S¹`{.Agda}, we show that this can not be the case. Suppose that is-iso is a proposition.
 
+[a proposition]: agda://1Lab.HLevel#is-prop
```
Comment (plt-amy) [inline src/1Lab/Counterexamples/IsIso.lagda.md:24]: your anchor is broken here

**Review by plt-amy — CHANGES_REQUESTED** [review]

File: src/1Lab/Counterexamples/IsIso.lagda.md (hunk 2 above)
Comment (jappeace) [reply src/1Lab/Counterexamples/IsIso.lagda.md]: Uh, I'm new to this, it's maybe fixed now?

**Comment by plt-amy** [comment]
Looks good, thanks!

**Pull request merged**